test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/MSBuildIntegrationTestBase.cs

```diff
@@ -24,9 +26,9 @@ protected MSBuildIntegrationTestBase()
 #error Configuration not supported
 #endif
 
-        protected string IntermediateOutputPath => Path.Combine("obj", Configuration, TargetFramework);
+        protected string IntermediateOutputPath => Path.Combine(Project.DirectoryPath, "obj", Configuration, TargetFramework);
```

Comment:
These aren't really intended to be full paths, or at least it isn't important that they are.

Reply:
It just ended up being repetitive. I can make the calling code specify it if you super hate this.

Reply:
No, you should handle it in all of the other cases. I want the APIs in the tests to be able to use relative paths - that's the part that gets repetative. Look at how Assert.File... works